src/pages/PlanPage/subRoutes/UpgradePlanPage/UpgradeDetails/UpgradeDetails.tsx

```diff
@@ -12,9 +12,15 @@ function UpgradeDetails({
 }: {
   selectedPlan: z.infer<typeof IndividualPlanSchema>
 }) {
-  if (selectedPlan?.value === Plans.USERS_SENTRYY) {
+  if (
+    selectedPlan?.value === Plans.USERS_SENTRYY ||
```

Comment:
There's utils for this that we could use, like the `isTeamPlan` or `isSentryPlan`